packages/nextra/src/server/rehype-plugins/twoslash/rehype-twoslash-popup.ts

```diff
@@ -0,0 +1,140 @@
+import { rendererRich, type RendererRichOptions } from '@shikijs/twoslash'
```

Comment:
can you move file `twoslash/rehype-twoslash-popup.ts` directly to `rehype-plugins` dir?